server/controllers/auth.js

```diff
@@ -47,9 +44,14 @@ exports.Login = async (req, res, next) => {
   Authenticate(req, res, next, 200);
 };
 
-exports.Self = async (req, res) => {
-  if (!req.user) {
-    return res.status(401).send();
-  }
-  return res.status(200).send(req.user);
+exports.Self = async (req, res) => res.status(200).send(req.user);
+
+exports.UpdateProfile = async (req, res) => {
+  const { terms, email, crawlers } = req.body;
+  const obj = { terms, email, crawlers };
```

Comment:
we should remove duplicates in terms right?

Reply:
Without giving insight to the user/front? 🤔 
I think we should warn the user about altering it's data.
I've added validation as a safety, but should be validated front first.

I've also removed duplicates from crawlers.